client/src/pages/Therapist/CreateProgram/AddDescription.js

```diff
@@ -45,15 +59,15 @@ const AddDescription = ({ navFunctions, setParentDescription, clientId }) => {
       validateForm();
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [description]);
+  }, [desc]);
```

Comment:
`useEffect` dependency array should take primitive data not object or array!
```suggestion
  }, [desc.title, desc.description]);
```